logstash-core/src/main/java/org/logstash/common/io/DeadLetterQueueWriter.java

```diff
@@ -131,7 +131,7 @@ private void innerWriteEntry(DLQEntry entry) throws IOException {
         byte[] record = entry.serialize();
         int eventPayloadSize = RECORD_HEADER_SIZE + record.length;
         if (currentQueueSize.longValue() + eventPayloadSize > maxQueueSize) {
-            logger.error("cannot write event to DLQ: reached maxQueueSize of " + maxQueueSize);
+            logger.error("cannot write event to DLQ(path:"+ this.queuePath +"): reached maxQueueSize of " + maxQueueSize);
```

Comment:
@amitavmohanty01, this looks pretty good to me. There are a couple code formatting issues, but those are pretty minor. The line above should include spaces in the places where I inserted underscores below:

`logger.error("cannot write event to DLQ_(path:_"_+ this.queuePath +_"): reached maxQueueSize of " + maxQueueSize);`

I'll need to defer to someone on the @elastic/logstash team to approve and merge this PR, though.

Reply:
I have modified as suggested and edited the commit.